backend/src/main/java/com/pickpick/service/MessageCreatedService.java

```diff
@@ -19,13 +22,17 @@ public class MessageCreatedService implements SlackEventService {
     private static final String TIMESTAMP = "ts";
     private static final String TEXT = "text";
     private static final String CLIENT_MSG_ID = "client_msg_id";
+    private static final String CHANNEL = "channel";
 
     private final MessageRepository messages;
     private final MemberRepository members;
+    private final ChannelRepository channels;
 
-    public MessageCreatedService(final MessageRepository messages, final MemberRepository members) {
+    public MessageCreatedService(final MessageRepository messages, final MemberRepository members,
+                                 final ChannelRepository channels) {
```

Comment:
취향 차이이긴 하지만, 이왕 줄바꿈 하는거 필드 하나마다 줄바꿈 하는건 어떤가 싶어요! 그냥 의견입니다~~
```suggestion
    public MessageCreatedService(
            final MessageRepository messages, 
            final MemberRepository members,
            final ChannelRepository channels
  ) {
```